lamindb/models/_from_values.py

```diff
@@ -345,50 +339,84 @@ def _bulk_create_dicts_from_df(
     return df.reset_index().to_dict(orient="records"), multi_msg
 
 
-def _has_organism_field(registry: type[Record]) -> bool:
+def _require_organism(registry: type[Record]) -> bool:
+    """Check if the registry has an organism field and is required.
+
+    Returns:
+        True if the registry has an organism field and is required, False otherwise.
+    """
     try:
-        registry._meta.get_field("organism")
-        return True
+        organism_field = registry._meta.get_field("organism")
+        # organism is not required or not a relation
+        if organism_field.null or not organism_field.is_relation:
+            return False
+        else:
+            return True
     except FieldDoesNotExist:
         return False
 
 
+def _is_simple_field_unique(field: FieldAttr) -> bool:
+    """Check if the field is an id field."""
+    # id field is a unique field that's not a relation
+    field = field.field
+    if field.unique and not field.is_relation:
+        return True
+    return False
+
+
 def _get_organism_record(  # type: ignore
```

Comment:
Why is this private?

Reply:
because it's a super low level function that nobody should use